posthog/hogql/test/test_property.py

```diff
@@ -520,17 +520,16 @@ def test_action_to_expr(self):
             event="$autocapture",
             action=action1,
             selector="a.nav-link.active",
-            tag_name="a",
+            tag_name="a",  # This is ignored!
```

Comment:
Worth also marking the field as deprecated on the model field, and TS interfaces, to prevent misuse

Reply:
Good idea. Marked it on the modal, but someone got there before me for TS: https://github.com/PostHog/posthog/blame/33d09c0db8c8938cd076b53d66d1ac81b45b2b95/frontend/src/types.ts#L512-L513